modules/appeals_api/app/models/concerns/appeals_api/higher_level_reviews/formatter.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module HigherLevelReviews
+    class Formatter
+      def initialize(higher_level_review)
```

Comment:
@nathanbwright What would you say to this parameter being something like

`AppealsApi::HigherLEvelReviews::FormDataAccess.new(higher_level_review)`